{
	"vulinfoid": "JVNDB-2017-012778",
	"vulinfodata": {
		"title": "389-ds-base におけるバッファエラーの脆弱性",
		"vulinfodescription": {
			"overview": "389-ds-base には、バッファエラーの脆弱性が存在します。"
		},
		"affected": {
			"affecteditem": [
				{
					"name": "Fedora Project",
					"productname": "389 Directory Server",
					"cpe": {
						"text": "cpe:/a:fedoraproject:389_directory_server",
						"version": "2.2"
					},
					"versionnumber": [
						"1.3.6.13 未満の 1.3.6.x",
						"1.3.7.9 未満の 1.3.7.x",
						"1.4.0.5 未満の 1.4.x"
					]
				},
				{
					"name": "レッドハット",
					"productname": "Red Hat Enterprise Linux",
					"cpe": {
						"text": "cpe:/o:redhat:enterprise_linux",
						"version": "2.2"
					}
				},
				{
					"name": "レッドハット",
					"productname": "Red Hat Enterprise Linux Desktop",
					"cpe": {
						"text": "cpe:/o:redhat:enterprise_linux_desktop",
						"version": "2.2"
					}
				},
				{
					"name": "レッドハット",
					"productname": "Red Hat Enterprise Linux Server",
					"cpe": {
						"text": "cpe:/o:redhat:enterprise_linux_server",
						"version": "2.2"
					}
				},
				{
					"name": "レッドハット",
					"productname": "Red Hat Enterprise Linux Workstation",
					"cpe": {
						"text": "cpe:/o:redhat:enterprise_linux_workstation",
						"version": "2.2"
					}
				}
			]
		},
		"impact": {
			"cvss": [
				{
					"version": "2.0",
					"severity": {
						"text": "Medium",
						"type": "Base"
					},
					"base": "5",
					"vector": "AV:N/AC:L/Au:N/C:N/I:N/A:P"
				},
				{
					"version": "3.0",
					"severity": {
						"text": "High",
						"type": "Base"
					},
					"base": "7.5",
					"vector": "CVSS:3.0/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H"
				}
			],
			"impactitem": {
				"description": "サービス運用妨害 (DoS) 状態にされる可能性があります。"
			}
		},
		"solution": {
			"solutionitem": {
				"description": "ベンダより正式な対策が公開されています。ベンダ情報を参照して適切な対策を実施してください。"
			}
		},
		"related": {
			"relateditem": [
				{
					"type": "vendor",
					"name": "Pagure",
					"vulinfoid": "CVE-2017-15134 389-ds-base: Remote DoS via search filters in",
					"url": "https://pagure.io/389-ds-base/c/6aa2acdc3cad9"
				},
				{
					"type": "vendor",
					"name": "Red Hat Bugzilla",
					"vulinfoid": "Bug 1531573",
					"url": "https://bugzilla.redhat.com/show_bug.cgi?id=1531573"
				},
				{
					"type": "vendor",
					"name": "Red Hat Security Advisory",
					"vulinfoid": "RHSA-2018:0163",
					"url": "https://access.redhat.com/errata/RHSA-2018:0163"
				},
				{
					"type": "advisory",
					"name": "Common Vulnerabilities and Exposures (CVE)",
					"vulinfoid": "CVE-2017-15134",
					"url": "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2017-15134"
				},
				{
					"type": "advisory",
					"name": "National Vulnerability Database (NVD)",
					"vulinfoid": "CVE-2017-15134",
					"url": "https://nvd.nist.gov/vuln/detail/CVE-2017-15134"
				},
				{
					"type": "cwe",
					"name": "JVNDB",
					"vulinfoid": "CWE-119",
					"url": "https://jvndb.jvn.jp/ja/cwe/CWE-119.html",
					"title": "バッファエラー"
				}
			]
		},
		"history": {
			"historyitem": [
				{
					"historyno": "1",
					"datetime": "2018-04-16T16:37:42+09:00",
					"description": "[2018年04月16日]\\n  掲載"
				}
			]
		},
		"datefirstpublished": "2018-04-16T16:37:42+09:00",
		"datelastupdated": "2018-04-16T16:37:42+09:00",
		"datepublic": "2017-10-08T00:00:00+09:00"
	}
}
